packages/office-ui-fabric-react/src/components/TextField/TextField.test.tsx

```diff
@@ -139,6 +139,12 @@ describe('TextField', () => {
     expect(suffixDOM.textContent).toEqual(exampleSuffix);
   });
 
+  it('should not give an aria-labelledby if no label is provided', () => {
```

Comment:
Thanks for adding this! I'd recommend adding one more test verifying that if the TextField has both a label and an `aria-labelledby` prop explicitly defined, it respects the prop instead of using the default generated label ID.